app/src/main/res/values/strings.xml

```diff
@@ -67,6 +67,10 @@
   <string name="play_pause_audio">Play/Pause Audio</string>
   <string name="help">Help</string>
   <string name="preferences">Preferences</string>
+  <string name="invalid_chars">Please only use numerical digits, spaces or forward slashes (/)</string>
```

Comment:
Here & for other errors: we should contextualize where these strings are used. E.g. fraction_error_invalid_chars.

Reply:
Done.